py-kms/pykms_Client.py

```diff
@@ -156,26 +156,34 @@ def client_check():
 def client_update():
         kmsdb = kmsDB2Dict()
 
+        loggerclt.debug(f'Searching in kms database for machine "{clt_config["mode"]}"...')
+
         appitems = kmsdb[2]
         for appitem in appitems:
                 kmsitems = appitem['KmsItems']
                 for kmsitem in kmsitems:                                
-                        name = re.sub('\(.*\)', '', kmsitem['DisplayName']).replace('2015', '').replace(' ', '')
```

Comment:
Why the removals in this file?

Reply:
To fix the discovery for `Windows 10` configuration in our file, which contains only `Windows 10/11` information.